### PR TITLE
Support workingDir and trim env

### PR DESCRIPTION
**Pull request by yahavi**
- [x] All [tests](https://github.com/jfrog/frogbot#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.

---

* Support new `JF_WORKING_DIR` environment variable to handle cases whereby the package descriptor is not in the root of the repository.
* Trim environment variables after reading them.

### CODE REVIEW

File: commands/scanpullrequest.go [hunk 1]
```diff
@@ -140,6 +141,9 @@ func auditSource(xrayScanParams services.XrayGraphScanParams, params *utils.Frog
 	if err != nil {
 		return []services.ScanResponse{}, err
 	}
+	if params.WorkingDirectory != "" {
```
Comment (talarian1) [inline commands/scanpullrequest.go:144]: We should probably do the same in the `auditTarget` func.

**Comment by talarian1** [comment]
I think we should consider adding a test project to this repo and an integration test for this scenario.

**Review by talarian1 — CHANGES_REQUESTED** [review]

File: templates/github-actions/frogbot-dotnet.yml [hunk 2]
```diff
@@ -45,6 +45,10 @@ jobs:
           # JFrog access token with 'read' permissions on Xray service
           # JF_ACCESS_TOKEN: ${{ secrets.JF_ACCESS_TOKEN }}
 
+          # [Optional, default: "."]
+          # Relative path in the repository to the project
```
Comment (eyalbe4) [inline templates/github-actions/frogbot-dotnet.yml:49]: Relative path in the repository to the project
-->
Relative path to the project in the git repository 

Let's make this change in all templates.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: commands/utils/params.go [hunk 3]
```diff
@@ -126,8 +128,20 @@ func extractInstallationCommandFromEnv(params *FrogbotParams) {
 	params.InstallCommandName = parts[0]
 }
 
+func readParamFromEnv(envKey string, paramValue *string) error {
```
[the remark this answers is not in the document]
Comment (yahavi) [reply commands/utils/params.go:131]: This logic returns an error:
```go
if *paramValue == "" {
	return &errMissingEnv{envKey}
}
```

**Pull request merged**